docs/rules/order-in-components.md

```diff
@@ -64,24 +64,26 @@ export default {
 If you want you can change the order providing the optional configuration in your `.eslintrc` file. Setting responsible for the above order looks like this:
 
 ```
-vue/order-in-components: [2, {
-  order: [
-    ['name', 'delimiters', 'functional', 'model'],
-    ['components', 'directives', 'filters'],
-    ['parent', 'mixins', 'extends', 'provide', 'inject'],
-    'el',
-    'template',
-    'props',
-    'propsData',
-    'data',
-    'computed',
-    'watch',
-    'lifecycle_hooks',
-    'methods',
-    'render',
-    'renderError'
-  ]
-}]
+{
+  'vue/order-in-components': [2, {
+    order: [
+      ['name', 'delimiters', 'functional', 'model'],
+      ['components', 'directives', 'filters'],
+      ['parent', 'mixins', 'extends', 'provide', 'inject'],
+      'el',
+      'template',
+      'props',
+      'propsData',
+      'data',
+      'computed',
+      'watch',
+      'LIFECYCLE_HOOKS',
+      'methods',
+      'render',
+      'renderError'
+    ]
+  }
+]
```

Comment:
`lifecycle_hooks` is invalid because it's should be `LIFECYCLE_HOOKS`

Its defined here [rules/order-in-components.js#L25](https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/rules/order-in-components.js#L25) and used [rules/order-in-components.js#L111](https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/rules/order-in-components.js#L111)

if you want to make it work we will have to change
```js
const extendedOrder = order.map(property => groups[property] || property)
```
to
```js
const extendedOrder = order.map(property => groups[property.toUpperCase()] || property)
```